### DISCUSSION TITLE
invitation

### DISCUSSION THREAD

**Discussion by trindadedev13** [Xed-Editor/Xed-Editor#142] [General]
Hello  @RohitKushvaha01 , Hi, I was testing your app and I saw that it is very good, congratulations, and well, we want to invite you to join the [Robok team](https://github.com/robok-inc/Robok-Engine). 
If you want to talk about it, and have telegram, access:  [Robok Chat](t.me/robokinc) 
Thank you, Aquiles Trindade.

**Comment by RohitKushvaha01** [comment]
> Hello  @RohitKushvaha01 , Hi, I was testing your app and I saw that it is very good, congratulations, and well, we want to invite you to join the [Robok team](https://github.com/robok-inc/Robok-Engine). 
> If you want to talk about it, and have telegram, access:  [Robok Chat](t.me/robokinc) 
> Thank you, Aquiles Trindade.  

I appreciate your invitation to join the Robok team.

While my experience with game engines is currently limited, I am confident that I can contribute in other areas of your project. I would be happy to discuss further and explore how my skills can align with your team's goals.

**Reply by trindadedev13** [reply]
I'm glad you accepted, welcome! Well, if possible, we can talk on Telegram.
